src/views/TransactionsCategories/IconPicker/IconPicker.js

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import { IconPickerItem } from "./IconPickerItem";
+import { iconList } from "../../../assets/iconList";
+import { IconPickerWrapper, IconWrapper } from "./IconPicker.style";
+
+export const IconPicker = ({ iconInputChangeHandler }) => {
+  return (
+    <IconPickerWrapper>
+      {iconList.map((icon) => (
```

Comment:
create one more component and pass iconList through props, right now you can not write unit test for this component, you have hard dependency

Reply:
done